modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/common/services/Quote.ts

```diff
@@ -28,3 +29,18 @@ export function getQuotes(parameters: Parameters = {}) {
 
 	return axios.get(`${DeliveryAPI}/`);
 }
+
+const adaptQuoteRequest = (state: any) => ({
```

Comment:
Try to type it correctly, if it's possible.

Reply:
Done